src/main/proto/service.proto

```diff
@@ -24,18 +24,22 @@ option java_outer_classname = "ServiceProto";
 
 service FinApp {
 
-  rpc CreateCustomer(Customer) returns (.google.protobuf.Empty) {}
+  rpc CreateCustomer(Customer) returns (CustomerResponse) {}
 
-  rpc CreateAccount(Account) returns (.google.protobuf.Empty) {}
+  rpc CreateAccount(Account) returns (AccountResponse) {}
 
-  rpc AddAccountForCustomer(CustomerRole) returns (.google.protobuf.Empty) {}
+  rpc AddAccountForCustomer(CustomerRole) returns (CustomerRoleResponse) {}
```

Comment:
How about renaming the AddCountForCustomer? I think we were discussing about:
CreateCustomerRoleRequest or something along these lines?
Will this be done in a separate PR?

Reply:
yes im going to do that in this PR, i just wanted to get the OK on everything i've changed so far before i change the name.

Reply:
Are we still changing the method name as part of this PR? I still see `AddAccountForCustomer`

Reply:
yes, im gonna start changing all the of the names (method names and message names)